app/design/frontend/Clothes/touch/Magento_Theme/templates/html/collapsible.phtml

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+?>
+
+<div class="block <?= $block->escapeHtmlAttr($block->getBlockCss()) ?>">
+    <div class="title <?= $block->escapeHtmlAttr($block->getBlockCss()) ?>-title"
+         data-mage-init='{"toggleAdvanced": {"toggleContainers": "#<?= $block->escapeHtmlAttr($block->getBlockCss()) ?>", "selectorsToggleClass": "active"}}'>
+        <strong>
+            <?= $block->escapeHtml(__($block->getBlockTitle())) ?>
+        </strong>
+    </div>
+    <div class="mag13-sidebar-test">MAG-13 content</div>
```

Comment:
@yuratouch Please, use `<p>`, or `<span>` tag if you want to add text in tag `<div>`